src/core/commandRunner/CommandRunner.ts

```diff
@@ -210,3 +220,20 @@ export default class CommandRunner {
     this.disposables.forEach(({ dispose }) => dispose());
   }
 }
+
+function constructThatTarget(
```

Comment:
This is now where the logic lives for wrapping selections in a `UntypedTarget`.  Seemed better than spreading it out